HPXMLtoOpenStudio/schematron_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'pathname'
```

Comment:
This file should be HPXMLtoOpenStudio/tests/test_schematron.rb and it should be structured like all the other test files (i.e., have a class that inherits from MiniTest::Test. And then you can have proper asserts in the tests.